src/components/IOBanner.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import PropTypes from 'prop-types';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import ioBanner from '../images/ioBanner.svg';
+
+const IOBanner = ({ isMobile }) => {
+  return (
+    <section
+      css={css`
+        position: relative;
+        margin-bottom: ${isMobile ? 0 : '2rem'};
+      `}
+    >
+      <img
+        css={css`
+          width: 100%;
+          @media (max-width: 500px) {
+            height: 150px;
+            object-fit: cover;
+            width: 100vw;
+          }
+        `}
+        src={ioBanner}
+        alt="New Relic IO"
+      />
+      <div
+        css={css`
+          position: absolute;
+          height: 0;
+          width: 45%;
+          color: var(--color-white);
+          margin: 0;
+          top: ${!isMobile ? '25%' : '15%'};
+          left: 5%;
+        `}
+      >
+        <h1
+          css={css`
+            color: var(--color-white);
+            font-size: ${isMobile ? '1rem' : '1.5vw'};
+          `}
+        >
+          Instant observability
+        </h1>
+        <p
+          css={css`
+            font-size: 0.75vw;
```

Comment:
I noticed that this scales the font super small at certain browser widths. Perhaps we can use [`max()`](https://developer.mozilla.org/en-US/docs/Web/CSS/max()):

```suggestion
            font-size: max(10pt, 0.75vw);
```

If we like that, we might want to update the heading too.

Reply:
ah yes, that's handy